WebJobs.Script.sln

```diff
@@ -160,6 +160,49 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "BlobTrigger-Batch", "BlobTr
 		sample\BlobTrigger-Batch\run.bat = sample\BlobTrigger-Batch\run.bat
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "EasyTables", "EasyTables", "{3EACC3F3-2747-42D6-B826-6A150F9EA92A}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "CSharp", "CSharp", "{9A295BFF-005D-456B-BBEA-6F4F389B7A66}"
+	ProjectSection(SolutionItems) = preProject
+		sample\EasyTables\CSharp\host.json = sample\EasyTables\CSharp\host.json
```

Comment:
I was trying to prevent every single sample from loading at once. So I nested mine into their own folder. Then it required a host.json.

This folder is going to explode if we're adding more samples -- I saw you had a different approach in one of your PRs where you list the folders you want to run (I think). I can adopt that pattern if that's the way we're going.

---
In reply to: [56186762](https://github.com/Azure/azure-webjobs-sdk-script/pull/98#discussion_r56186762) [](ancestors = 56186762)